docs/cross-language-functions.md

```diff
@@ -0,0 +1,98 @@
+---
+sidebar_position: 11
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Cross Language Functions
+
+Less facilitates the utilization of a function created in one programming language in another. This implies that you can use a Python function in JavaScript and vice versa.
+In order to create a cross language function, you will need to create a folder named `functions` in the less directory
+
+```bash
+mkdir -p less/functions
+```
+
+You can have multiple functions, for example I want to create a function that sum two numbers.
+```bash
+mkdir less/functions/sum
+```
+
+<Tabs>
+  
+  <TabItem value="nodejs" label="Node.js">
+    ```bash
+    touch less/functions/sum/index.js
+    ```
+    
+    ```js title="less/functions/sum/index.js" showLineNumbers
+    exports.process = (data) => {
+      return data.first_number + data.second_number;
+    }
+    ```
+  </TabItem>
+
+  <TabItem value="py" label="Python">
+    ```bash
+    touch less/functions/sum/__init__.py
+    ```
+
+    ```py title="less/functions/sum/__init__.py" showLineNumbers
+    def process(data):
+      return data.get('first_number') + data.get('second_number')
+    ```
+  </TabItem>
+  
+</Tabs>
+
+### Calling your cross language function
+
+<Tabs>
+  
+  <TabItem value="nodejs" label="Node.js">
+    Import `functions` from `@chuva.io/less` to call the function and process your payload in order to retrieve the response.
+
+    ```js showLineNumbers
+    const { functions } = require('@chuva.io/less');
+    ```
+
+    Now you can call your Python function on Javascript
+
+    Let's sum two numbers
+    ```jsx showLineNumbers
```

Comment:
@RicardoJRR-23 Is there any reason to have `js showLineNumbers` instead of `jsx showLineNumbers`?